test/test_bind_integer.rb

```diff
@@ -0,0 +1,41 @@
+require 'oci8'
+require File.dirname(__FILE__) + '/config'
+
+class TestBindTime < Minitest::Test
```

Comment:
Ah, sure... One more confirmation that copy/paste is evil :-)

I'm also going to:
- [ ] add tests with `Integer` as output parameter
- [ ] Fix the misleading comment of cursor.bind_param about unsupported Bignum, etc.